README.md

```diff
@@ -524,15 +524,17 @@ strategies:
 
 This strategy evicts pods that are older than `maxPodLifeTimeSeconds`.
 
-You can also specify `podStatusPhases` to `only` evict pods with specific `StatusPhases`, currently this parameter is limited
-to `Running` and `Pending`.
+You can also specify `states` parameter to **only** evict pods matching the following conditions:
+  - [Pod Phase](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase) status of: `Running`, `Pending`
+  - [`ContainerStateWaiting`](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-states) condition of: `PodInitializing`, `ContainerCreating`
```

Comment:
Is this supposed to say `ContainerStateWaiting` or `ContainerState`? The link talks about Waiting, Running, and Terminating states. Maybe this section could be clarified what the states are, feel free to add more description outside of the bullet points

Reply:
Currently PodLifeTime will only filter on `Reasons` of `State.Waiting`

Code [here](https://github.com/kubernetes-sigs/descheduler/pull/834/files#diff-4b7a41236384b65d820a644397e833df40452f002d63e2ade2b145904fe04170R101)

Reply:
good catch on the link, updated